v2.1/reference/sql/constraints.md

```diff
@@ -45,14 +45,18 @@ ALTER TABLE orders DROP FOREIGN KEY fk_user_id;
 ALTER TABLE orders ADD FOREIGN KEY fk_user_id (user_id) REFERENCES users(id);
 ```
 
-然而，TiDB 不会在 DML 语句中对外键进行约束。例如，即使 `users` 表中不存在 `id=123` 的记录，下列事务也能提交成功：
+### 注意
+
+* TiDB 不会在 DML 语句中对外键进行约束。例如，即使 `users` 表中不存在 `id=123` 的记录，下列事务也能提交成功：
 
 ```
```

Comment:
Since you used a list, please indent the code block with 4 spaces for each line. This is to make the display better organized.